### ISSUE TITLE
Ember CLI Deploy Failing on Windows Due to ESLint

### ISSUE THREAD

**Issue by jamesdixon** [ember-cli/ember-cli-eslint#58]
When running `ember deploy`, I'm seeing the following error:

```
Invalid regular expression: /(\)?build(\)?$/: Unterminated group
SyntaxError: Invalid regular expression: /(\)?build(\)?$/: Unterminated group
    at new RegExp (native)
    at Object.<anonymous> (C:\projects\app-business\node_modules\broccoli-lint-eslint\build\index.js:10:23)
    at Module._compile (module.js:413:34)
    at Object.Module._extensions..js (module.js:422:10)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Module.require (module.js:367:17)
    at require (internal/module.js:16:19)
    at Object.<anonymous> (C:\projects\app-business\node_modules\ember-cli-eslint\index.js:2:14)
    at Module._compile (module.js:413:34)
```

Is there a way to disable linting at this stage?

Thanks!

**Comment by morhook** [comment]
I think this might be related with a modification done for running on Windows. Are you running on one of those or unix-based?

**Comment by jamesdixon** [comment]
You're right on. I'm using a Windows-based CI system and that's where it's failing.

**Comment by BrianSipple** [comment]
There's a [PR for a change in `broccoli-lint-eslint` that should fix this](https://github.com/ember-cli/broccoli-lint-eslint/pull/36).

I'm going to merge it now and bump the patch version -- `ember-cli-eslint` should receive that.

**Comment by jamesdixon** [comment]
Thanks, @BrianSipple! Much appreciated 👍

**Comment by BrianSipple** [comment]
@jamesdixon Just published `broccoli-lint-eslint@2.2.1`. 

An `rm -rf node_modules && npm cache clean && npm install` should get the latest to your app using `ember-cli-eslint`

**Comment by jamesdixon** [comment]
Works like a charm. Thanks again!
